src/server/system_services/stats_aggregator.js

```diff
@@ -459,10 +459,21 @@ async function _partial_buckets_info(req) {
 
             const OPTIMAL_MODES = [
                 'LOW_CAPACITY',
-                'HIGH_DATA_ACTIVITY',
+                'DATA_ACTIVITY',
```

Comment:
bucket_info.mode doesn't have such a value.
You should change that in the pool_mode schema as well.